gap/perm.gi

```diff
@@ -16,19 +16,19 @@
 ##
 #############################################################################
 
-#! @BeginChunk VeryFewPoints
-#! If a permutation group acts only on a few points (the current limit is at
-#! most 10 points) then a stabiliser chain is computed by the randomized &GAP;
-#! library function for that purpose. If the method is successful then the
+#! @BeginChunk VerySmallPoints
```

Comment:
Perhaps something like `MovesOnlySmallPoints` or a variation thereof is even more self-explaining?

Reply:
I like it.